corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js

```diff
@@ -88,6 +88,9 @@ hqDefine("cloudcare/js/formplayer/menus/controller", function () {
 
         if (menuResponse.breadcrumbs) {
             hqImport("cloudcare/js/formplayer/menus/util").showBreadcrumbs(menuResponse.breadcrumbs);
+            if (menuResponse.langs) {
```

Comment:
Should this check `menuResponse.langa.length > 1`?

Reply:
yeah, I can do that.